src/Serilog.Sinks.RollingFile/Sinks/RollingFile/RollingFileSink.cs

```diff
@@ -39,11 +39,12 @@ public sealed class RollingFileSink : ILogEventSink, IDisposable
         readonly int? _retainedFileCountLimit;
         readonly Encoding _encoding;
         readonly bool _buffered;
+        readonly bool _shared;
         readonly object _syncRoot = new object();
 
         bool _isDisposed;
         DateTime? _nextCheckpoint;
-        FileSink _currentFile;
+        ILogEventSink _currentFile;
```

Comment:
Ideally the two file sinks will expose a common interface/ABC; for now `ILogEventSink` is the best I could do here.